{{cookiecutter.app_name}}_role/molecule/default/Dockerfile.j2

```diff
@@ -8,5 +8,5 @@ FROM {{ item.image }}
 
 # install minimal packages for debian slim images
 RUN apt-get update && \
-    apt-get install -y python sudo bash ca-certificates iproute2 systemd systemd-sysv python-pip && \
+    apt-get install -y python sudo bash ca-certificates iproute2 systemd systemd-sysv python3-pip && \
```

Comment:
why not `python3`?

Reply:
I did not check that, remove python2 and add python3 to Dockerfile.